dashboard/app/models/plc/README.md

```diff
@@ -0,0 +1 @@
+Explanation of PLC Model and objects is here - http://wiki.code.org/display/Operations/Explanation+of+PLC+Model

```

Comment:
Given that contributors and others cannot access our internal wiki, does it make sense to move the contents of the wiki page into this file?